src/features/encoders/avif/client/index.tsx

```diff
@@ -183,7 +190,7 @@ export class Options extends Component<Props, State> {
             <div class={style.optionOneCell}>
               <Range
                 min="0"
-                max="63"
+                max="99"
```

Comment:
I know this wasn’t the case previously, but it really should have been 😅 

```suggestion
                max={maxQuality - 1}
```

Same below!

Reply:
Yep that'd be easier to read, on it!